src/commonMain/kotlin/exchange.dydx.abacus/state/manager/StateManagerAdaptor.kt

```diff
@@ -1410,9 +1410,9 @@ open class StateManagerAdaptor(
             url,
             historicalTradingRewardsInPeriod,
             "startedAt",
-            0.days,
-            180.days,
-            "endedAt",
+            1.days,
+            180.days, // OTE-337: update this to read in the trading rewards start date from env.json
```

Comment:
this works for now (because earliest date should be Jan 10/29 depending on whether we're in test or mainnet), but should be calculated from the env.json start date which is still in progress. Will update if that makes it in before this makes it in, otherwise will make new PR